### DISCUSSION TITLE
How to style section number for differents sections ? (h1, h2, etc.) ?

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#7925] [Q&A]
### Description

I use this css rule to style my sections numbers :

    .header-section-number {
      color: #345 !important;
     }

How can I set a style section number for differents sections ? (h1, h2, etc.) ? 

Something like : 

    .header-section-h1-number {
      color: #0000 !important;
     }

    .header-section-h2-number {
      color: #fffff !important;
     }

**Comment by cderv** [comment]
Best way would be for you to 

* inspect the HTML document (using chrome or any browser), 
* find the Elements you want to style
* Add to your CSS your styling the right CSS selectors 

Note that Quarto offers HTML theming feature through variables and theme files. Changing the font size of headers can be done easily using a SASS variable: https://quarto.org/docs/output-formats/html-themes.html#fonts
